opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -362,15 +362,15 @@ enum AggregationTemporality {
 // enum is a bit-mask.  To test the presence of a single flag in the flags of
 // a data point, for example, use an expression like:
 //
-//   (point.flags & FLAG_NO_RECORDED_VALUE) == FLAG_NO_RECORDED_VALUE
+//   (point.flags & DATA_POINTS_FLAGS_NO_RECORDED_VALUE) == DATA_POINTS_FLAGS_NO_RECORDED_VALUE
 //
 enum DataPointFlags {
-  FLAG_NONE = 0;
+  DATA_POINTS_FLAGS_NONE = 0;
```

Comment:
This name change seems 100% inline w/ C++ generated code (vs. other language).  Is that correct?  Do we have a naming convention for enums that requires the value name to include the enum name?